packages/data-service-generator/src/create-data-service.ts

```diff
@@ -47,6 +47,8 @@ export async function createDataService(
       durationInMs: endTime - startTime,
     });
 
+    await context.logger.info("App generation process finished successfully");
```

Comment:
you can use `logger.info` instead of context and you don't need sync method for that